operator/pkg/controllers/master/kubescheduler.go

```diff
@@ -85,6 +81,10 @@ func schedulerPodSpecFor(controlPlane *v1alpha1.ControlPlane) v1.PodSpec {
 					v1.ResourceCPU: resource.MustParse("1"),
 				},
 			},
+			Ports: []v1.ContainerPort{{
+				ContainerPort: 10251,
```

Comment:
same problem. What happen if we choose to use KubernetesVersion=v1.21?

Reply:
In 1.21 KCM and scheduler is not yet deprecated